packages/api-contract/src/MetaRegistry.ts

```diff
@@ -0,0 +1,333 @@
+// Copyright 2017-2019 @polkadot/api-contract authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import * as t from '@polkadot/types/types';
```

Comment:
Import explicitly what we are using, i.e. `import { MetaRegistryJson, ... } from '@polkadot/types/types';`

Reply:
Still needs an adjustment.